src/olympia/constants/search.py

```diff
@@ -21,7 +21,7 @@
     'danish': ['da'],
     'german': ['de'],
     'greek': ['el'],
-    'english': ['en-us'],
+    'english': ['en-us', 'en-gb'],
```

Comment:
My main concern was when returning the strings back for the API but then I remembered it goes into a different, specific field that is not analyzed so it's fine :)